### ISSUE TITLE
Taskbarx losing blur, needs restart to fix

### ISSUE THREAD

**Issue by ion-elgreco** [ChrisAnd1998/TaskbarX#284]
Store or Regular version: Store version
What's your Build of Windows 10: Build 18362.19h1_release.190318-1202
.NET Framework 4.8 installed: Yes
Any Antivirus software: No
Any other third-party taskbar tweakers: No
Is there a way i can reproduce: No
What are your taskbar settings: Only speed change to 100ms
What says the error message if you got any: No error message

Taskbar loses its blur randomly and then I can only get it back after a force quit from configuratorx.

**Comment by JPHFanedits** [comment]
I am also experiencing a similar issue. The blur and acrylic features do not work at all and both function as transparent. I am unable to get them to work even after restart of app and computer. However, if I click on the start button it seems to restore blur and acrylic functions

**Comment by FoxesPaws** [comment]
I can also confirm this, I also had this problem with a rainmeter-transparent taskbar skin as well (I'm no longer using rainmeter-transparent taskbar skin). The blur disappears completely and goes fully transparent sometimes. So far I've fixed it by restarting TaskbarX. 

What's your Build of Windows 10: Build 18363.836
.NET Framework 4.8 installed: Yes
Any Antivirus software: Yes
Any other third-party taskbar tweakers: No
Is there a way I can reproduce: No
What are your taskbar settings: Only speed change to 500ms, Animationspeed changed to 250ms
What says the error message if you got any: No error message

**Comment by ion-elgreco** [comment]
You guys should try out 1.5.7. It has fixed this issue for me. https://github.com/ChrisAnd1998/TaskbarX/releases/tag/1.5.7.0

**Comment by FoxesPaws** [comment]
> 
> 
> You guys should try out 1.5.7. It has fixed this issue for me. https://github.com/ChrisAnd1998/TaskbarX/releases/tag/1.5.7.0

Just uninstalled the old release and installed this, still the same problem. Although I've found a way to possibly reproduce the problem. If windows taskbar settings are set to "Automatically hide the taskbar in desktop mode" = true, then by switching on and off with "Use small taskbar buttons" (note: just spam that switch-button) I was able to make the problem both returning and going away.
